src/odoc/page-index.mld

```diff
@@ -0,0 +1,252 @@
+{1:top odoc}
+
+The documentation compiler for OCaml, Reason, and BuckleScript.
+
++ {{:#overview} What is [odoc] really? }
++ {{:#flow} How does it work? }
++ {{:#deep-dive} Digging Deeper }
+
+{2:overview What is [odoc] really?}
+
+[odoc] is not your regular documentation tool.
+
+Instead of source code, it works with {i compilation units}; that is, it works
+on compiler outputs and turns them into compiled documentation, which then
+becomes gorgeous HTML.
+
+A regular [odoc] execution transforms [cmt], [cmti], and [mld] files into [odoc]
```

Comment:
Maybe you can explain what `odoc` files are before mentioning them here? Surely people will ask "what is an odoc file?".